pkgs/development/libraries/libdicom/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, uthash, meson, ninja, cmake, check, pkg-config }:
+
+stdenv.mkDerivation rec {
```

Comment:
I think it would be useful to use the `finalAttrs` pattern here
* History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293/12
* Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes

(also see other places in nixpkgs, more and more packages adopt it)

Reply:
Very interesting! Thanks!